.github/workflows/release.yml

```diff
@@ -0,0 +1,18 @@
+name: android-release
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Might it be helpful to have this also run on PRs against main?  If it were actually making a release I see why this would not be desirable but right now it seems like it is only uploading the APK to GitHub Actions artifact storage, right?  I think it would be useful to have easy access to APKs for PRs before they are merged.